ui/main/src/app/business/view/navbar/navbarMenu.view.ts

```
@@ -17,13 +17,18 @@ import {RouterStore} from 'app/business/store/router.store';
-    private navbarMenu: NavbarMenu;
-    private destroy$ = new Subject<void>();
+    private static readonly ADMIN_MENUS = ['admin', 'externaldevicesconfiguration'];
+    private static readonly VISIBLE_RIGHT_MENUS_IN_COLLAPSED_MODE = [
+        'settings',
+        'feedconfiguration',
+        'nightdaymode',
+        'logout'
+    ];
+    private readonly navbarMenu: NavbarMenu;
+    private readonly destroy$ = new Subject<void>();
     private menuChangeListener: Function;
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)